plugins/manager/pages/tableset_import.php

```diff
@@ -13,6 +13,7 @@
 $yform = new rex_yform;
 $yform->setHiddenField('page', $page);
 $yform->setObjectparams('real_field_names',true);
+$yform->setObjectparams('main_id', 'tableset_import');
```

Comment:
man könnte es auch weglassen, aber dann wird der Dateiname mit "-1" prefixed was man dann in Zeile 40 und 48 voranstellen müsste.
Kommt mir so leserlicher vor (könnte man auch als variable definieren)